egs2/TEMPLATE/hubert_asr1/scripts/utils/TEMPLATE_Readme.md

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
Are you going to add a proper README?
There's a lot going on here!  I see config files with tons of datasets.  These are all used for pretraining, I guess?
Would be nice to have a README explaining what's going on.

Reply:
Good idea! Sure I will add it.